chapters/es/slides/chapter1_02_statistical-models.md

```diff
@@ -66,7 +66,7 @@ import spacy
 nlp = spacy.load("es_core_news_sm")
 
 # Procesa un texto
-doc = nlp("Ella comió pizza")
+doc = nlp("Ella comió pizza.")
```

Comment:
Are you sure this is correct? The example tokens below don't have the `.` and I think for simplicity, I left it out in the other languages as well.